client/src/views/Simulation/components/SimulationLegend.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <ul class="legend-simulation">
+    <li class="run current">Current Run</li>
+    <li class="run previous">Previous Runs</li>
+    <li class="run aggregate">Aggregate of all Runs</li>
```

Comment:
We probably want to add here the line for observational data (even if we don't have it yet). I think the color should be `#E75BCD` but @emilykuang can confirm.

Reply:
Thanks, I was thimking that 3 legend buttons would be too much.

Do you think we need subtitles for the legend? I'm worriewd that user might think the runs colours are for edges. @emilykuang what are your thoughts?

Reply:
Thanks for adding the observational data. Yes, the subtitles are good for differentiating between the Model Panel and Variable Panel, the divider looks great! 
I am just wondering about the wording for the yellow line: the meaning of `Aggregate` is "calculated by the combination of units; total" which may be confusing, many a better word is `Average`?

Reply:
100%

Reply:
![Screen Shot 2021-08-11 at 09 30 05](https://user-images.githubusercontent.com/636801/129037822-9b4d788e-cc89-4dbf-a90c-a052f1fd91df.png)